apps/forms/templates/widgets/plea_radio_field.html

```diff
@@ -5,90 +5,131 @@
+
+                    <a class="details-trigger" href="#charge-details">
+                        <span class="summary">
+                            {% blocktrans %}View charge information{% endblocktrans %}
+                        </span>
+                    </a>
+
+                    <div class="details-content" id="charge-details">
+                        {% if LANGUAGE_CODE == "cy" and form.case_data.offence_wording_welsh %}
+                            {{ form.case_data.offence_wording_welsh|linebreaks }}
+                        {% else %}
+                            {{ form.case_data.offence_wording|linebreaks }}
+                        {% endif %}
+                    </div>
                 </div>
-            </div>
 
                 {% if plea_data.split_form != "split_form_last_step" %}
-            <h3 class="heading-small">{% blocktrans %}Your plea for this charge{% endblocktrans %}</h3>
+                    <h3 class="heading-small">
+                        {% blocktrans %}Your plea for this charge{% endblocktrans %}
+                    </h3>
                 {% endif %}
+
             {% endif %}
 
             {% if form.guilty.errors %}
-            <ul class="errorlist">
-                {% for error in form.guilty.errors %}
-                <li>{{ error }}</li>
-                {% endfor %}
-            </ul>
+                <ul class="errorlist">
+                    {% for error in form.guilty.errors %}
+                    <li>{{ error }}</li>
+                    {% endfor %}
+                </ul>
             {% endif %}
+
         </legend>
 
         {% if plea_data.split_form != "split_form_last_step" %}
-        <div class="form-field">
-            <div class="inline">
 
-                {% for choice in form.guilty %}
-                <label for="id_{{ choice.name }}_{{ choice.choice_value }}" class="block-label">
-                    <input id="id_{{ choice.name }}_{{ choice.choice_value }}" type="radio" name="{{ choice.name }}" value="{{ choice.choice_value }}" {% if choice.is_checked %}checked="checked"{% endif %}>
-                    {{ choice.choice_label }}
-                </label>
-                {% endfor %}
+            <div class="form-field">
+                <div class="inline">
+
+                    {% for choice in form.guilty %}
+                        <label for="id_{{ choice.data.name }}_{{ choice.data.value }}"
+                               class="block-label">
+                            <input id="id_{{ choice.data.name }}_{{ choice.data.value }}"
+                                   type="radio"
+                                   name="{{ choice.data.name }}"
+                                   value="{{ choice.data.value }}"
+                                   {% if choice.data.selected %}checked="checked"{% endif %}>
+                            {{ choice.choice_label }}
+                        </label>
+                    {% endfor %}
```

Comment:
Done. Well, I'm not sure the template's not used but the form now just uses dsradioselect.